### ISSUE TITLE
ollama support

### ISSUE THREAD

**Issue by ToeiRei** [matrixgpt/matrix-chatgpt-bot#304]
As far as I know, ollama is working on [OpenAI compatibility](https://github.com/ollama/ollama/blob/main/docs/openai.md).

To my understanding, changing the host to use your local ollama instance should work... (have not tested yet)

**Comment by philidinator** [comment]
Hi, I'm also interested, did you try it out yet?

**Comment by ToeiRei** [comment]
No, I did not spoof the endpoint with /etc/hosts yet as I run a more complex setup here.

**Comment by philidinator** [comment]
I tried setting CHATGPT_REVERSE_PROXY=http://ip:11434/v1/chat/completions, but unfortunately it didn't work. I always get an error message: [ERROR] [OpenAI-API Error: Error: Failed to send message. HTTP 400 - {"error":{"message":"[] is too short - 'messages'","type":"invalid_request_error","param":null,"code":null}}].
